Task7.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+
+class LoginForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      errors: {
+        email: '',
+        password: '',
+      },
+    };
+  }
+
+  handleChange = (e) => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    const { email, password } = this.state;
+    const errors = {};
+
+    // Email validation (simple check for a valid email format)
+    if (!email.match(/^[\w-]+(\.[\w-]+)*@([\w-]+\.)+[a-zA-Z]{2,7}$/)) {
```

Comment:
This is good enough but the regex expression for mail validation can be more crisp like below may be 
'/^[a-z0-9]+@[a-z]+\.[a-z]{2,3}$/;' we might need to change our regex as per the requirement but this is fine.

Refer this one to understand what is the meaning of it. -> https://www.tutorialspoint.com/how-to-validate-email-address-using-regexp-in-javascript